tests/cypress/e2e/unit_tests/p0_fleet.spec.ts

```diff
@@ -19,6 +19,7 @@ export const appName = "nginx-keep";
 export const clusterName = "imported-0";
 export const branch = "main";
 export const path  = "nginx"
+export const sshString = ["Public key and private key for SSH", "Public key and private key for SSH authentication"]
```

Comment:
Currently I see this used solely in one place.
Given this perhaps had made more sense to place it in the specific test where it is called to ease its location / use rather than as a general export. No need to change it now. Just a suggestion for later pr's.

Reply:
Okay, I will update it in next PR. Thanks for the suggestion.